ligo/stablecoin/spec.ligo

```diff
@@ -215,12 +215,19 @@ type user_permits is
 
 type permits is big_map (address, user_permits)
 
-type permit_param is (key * (signature * blake2b_hash))
+type permit_signature is michelson_pair(signature, "", blake2b_hash, "permit_hash")
```

Comment:
This change simply adds an annotation, right?

Reply:
Correct, it just adds `permit_hash` to match the parameter definition in `docs/specification.md`:
```
(pair %permit
  key
  (pair
    signature
    (bytes %permit_hash))
)
```